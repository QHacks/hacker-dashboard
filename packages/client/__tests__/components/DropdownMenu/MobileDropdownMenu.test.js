import React from "react";
import MobileDropdownMenu from "../../../components/DropdownMenu/MobileDropdownMenu";
import { shallow } from "enzyme";

describe("MobileDropdownMenu", () => {
  it("renders correctly", () => {
    expect(shallow(<MobileDropdownMenu />)).toMatchSnapshot();
  });
});
