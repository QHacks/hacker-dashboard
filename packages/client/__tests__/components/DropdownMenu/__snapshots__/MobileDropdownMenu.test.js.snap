// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobileDropdownMenu renders correctly 1`] = `
<div
  className="css-8atqhb"
>
  <button
    onClick={[Function]}
  >
      
    â–¼
  </button>
</div>
`;
